deploy/scripts/yaml_helpers.py

```
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
```

Ruff (0.14.0)

Shebang is present but file is not executable

(EXE001)